bin/prefix-vendor-namespace.php

```diff
@@ -83,6 +88,16 @@
 	'psr/container'    => [
 		'league/container',
 	],
+	'psr/http-message' => [
+		'firebase/php-jwt',
+		'google/apiclient',
+		'google/auth',
+		'google/gax',
+		'guzzlehttp/guzzle',
+		'guzzlehttp/psr7',
+		'psr/http-client',
+		'psr/http-factory',
```

Comment:
I think we need to prefix both `psr/http-message` and `psr/http-factory`. But I don't think this belongs as a dependency. 

We aren't using any of the factory classes directly so it doesn't seem to break anything. But since they share a namespace it's causing some replacements which could cause failures.

For example in `vendor/guzzlehttp/psr7/src/HttpFactory.php` we are prefixing all instances of `Psr\Http\Message`. But for the class `Psr\Http\Message\RequestFactoryInterface`, the class can be found in `vendor/psr/http-factory/src/RequestFactoryInterface.php` but doesn't use a prefixed namespace. Same thing for other factory interfaces.